lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -3,6 +3,9 @@
  * Supports HTML5 and Flash.
  */
  var edx = edx || {};
+ var key = {
+    enter: 13
+ };
 
```

Comment:
Duplicate var calls. You can use commas to separate var declarations so the code can be updated to 
```
var edx = edx || {},
      key = {
           enter: 13
      };
```